packages/corvid-local-testkit/src/index.js

```diff
@@ -1,33 +1,37 @@
 "use strict";
+const { localSiteDir } = require("corvid-local-test-utils");
+const corvidLocalServerTestkit = require("corvid-local-server/src/testkit");
 
-var fs = require("fs");
-var path = require("path");
+const LOCAL_SITE_PATH = Symbol.for("local-site-path");
 
-var { initTempDir } = require("corvid-local-test-utils");
-var corvidLocalServerTestkit = require("corvid-local-server/src/testkit");
-
-function initSite(siteItems) {
-  return initTempDir(siteItems).then(function(localSite) {
-    fs.writeFileSync(
-      path.join(localSite, ".corvid", "corvidrc.json"),
-      "",
-      "utf8"
+const server = {
+  startInEditMode: function(localSite) {
+    return corvidLocalServerTestkit.startInEditMode(localSite[LOCAL_SITE_PATH]);
+  },
+  startInCloneMode: function(localSite) {
+    return corvidLocalServerTestkit.startInCloneMode(
+      localSite[LOCAL_SITE_PATH]
     );
-    return Promise.resolve(localSite);
-  });
-}
+  }
+};
 
-var server = {
-  startInEditMode: function(siteItems) {
-    return corvidLocalServerTestkit.startInEditMode(siteItems);
-  },
-  startInCloneMode: function(siteItems) {
-    return corvidLocalServerTestkit.startInCloneMode(siteItems);
+const localSite = {
+  init: async files => {
+    const tempDir = await localSiteDir.initLocalSite(files);
+    return {
+      [LOCAL_SITE_PATH]: tempDir,
+      getFiles: () => localSiteDir.readLocalSite(tempDir),
+      readFile: filePath => localSiteDir.readFile(tempDir, filePath),
+      writeFile: (filePath, content) =>
+        localSiteDir.writeFile(tempDir, filePath, content),
+      deleteFile: filePath => localSiteDir.deleteFile(tempDir, filePath),
+      clean: () => localSiteDir.cleanLocalSite(tempDir)
+    };
   }
 };
 
-var localTestkit = {
-  initSite,
+const localTestkit = {
```

Comment:
no real need to the "localTestkit" variable. just exports the functions

Reply:
done